### DISCUSSION TITLE
Git 分支管理与版本发布 | 夜天之书

### DISCUSSION THREAD

**Discussion by giscus** [tisonkun/blog#89] [Comments]
# Git 分支管理与版本发布 | 夜天之书

Git 版本管理系统由 Linux 的作者 Linus Torvalds 于 2005 年创造，至今不到二十年。 起初，Git 用于 Linux Kernel 的协同开发，用于替代不再提供免费许可的 BitKeeper 软件。随后，这一提供轻量级分支的分布式版本管理系统得到了开源开发者的广泛喜爱，在大量开源项目中投入使用。如今，Git 几乎是版本管理系统的同义词。 Git 最大的创新就是轻量级的分

https://www.tisonkun.org/2023/05/19/git-branch-and-release/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

这其实约等于 DDX ~ 开发者文档体验 的问题,
无论使用哪种版本管理系统,
一个工程的版本管理流程本身就是工程的一部分,
应该一开始就清晰明了的约定好, 并长期执行下去,
除非发现重大问题, 也得公开说明清楚后变更...

不过,  GitHub 们强力统一江湖后, 问题就简化为使用 gh 的哪种功能...
可能, 也是个潜在的问题...
